src/routes/InventoryTable.js

```diff
@@ -290,6 +324,22 @@ const Inventory = ({
                     onEditOpen(false);
                 }}
             />
+            <AddHostToGroupModal
+                isModalOpen={addHostGroupModalOpen}
+                setIsModalOpen={setAddHostGroupModalOpen}
+                modalState={{ id: 1, name: 'name', groupName: 'groupName' }}
+                reloadData={() => console.log('data reloaded')}
+                setIsCreateGroupModalOpen={setIsCreateGroupModalOpen}
+                //deviceIds should pass the id
+                deviceIds={['123']}
+            />
+            {isCreateGroupModalOpen && (
+                <CreateGroupModal
+                    isModalOpen={isCreateGroupModalOpen}
+                    setIsModalOpen={setIsCreateGroupModalOpen}
+                    reloadData={() => console.log('data reloaded')}
+                />
+            )}
```

Comment:
Can we move the render of CreateGroupModal inside of AddHostToGroupModal? This should make code a bit cleaner and move the show/not-show boolean logic inside the children component, not InventoryTable. Less state variables on the high-level component - the better and easier to maintain.

Reply:
I updated the PR accordingly